networks/suzuka/suzuka-full-node/src/main.rs

```diff
@@ -8,6 +8,7 @@ use std::process::ExitCode;
 fn main() -> Result<ExitCode, anyhow::Error> {
 	let runtime = tokio::runtime::Builder::new_multi_thread().enable_all().build()?;
 
+	// console_subscriber::init();
```

Comment:
This could be composed with `tracing_subscriber` and conditionally enabled. The docs give an [example](https://docs.rs/console-subscriber/latest/console_subscriber/#adding-the-console-subscriber) to build on.

Reply:
Yes, I needed to figure out how to add the `EnvFilter` though and deprioritized.